app/auth.py

```
@@ -0,0 +1,223 @@
+    """Handle the Auth0 callback - get and store tokens, log user in"""
+    current_app.logger.error(f"=== CALLBACK START ===")
+    current_app.logger.error(f"Session at callback: {dict(session)}")
+    current_app.logger.error(f"Session permanent: {session.permanent}")
+    current_app.logger.error(f"Login timestamp from session: {session.get('login_timestamp', 'MISSING')}")
+    
+    # Get the state parameter from the request
+    state_from_request = request.args.get('state')
+    current_app.logger.error(f"State from Auth0 callback: {state_from_request}")
+    
+    try:
+        token = auth0.authorize_access_token()
```

Ruff (0.11.9)

Local variable `token` is assigned to but never used

Remove assignment to unused variable `token`

(F841)